library/Zippy/ZipArchive.hpp

```diff
@@ -37,6 +37,10 @@
 #include <memory>
 #include <fstream>
 
+#ifdef _WIN32
+#include <direct.h>
```

Comment:
```suggestion
#include <direct.h>
#else
#include <sys/types.h>
#include <sys/stat.h>
#include <unistd.h>
```

Currently Zippy also fails to build under *nix, with certain libc implementations that doesn't get you `mkdir()` with the above includes. This fixes the issue.

I know this isn't with the title, but since another PR would likely cause a merge conflict when either one eventually gets accepted (adding the `same #ifdef _WIN32`). Thank you very much if you could change the title and accept this suggestion.

Reply:
Sure